requirements/adjust_versions.py

```diff
@@ -14,7 +14,6 @@
     dict(torch="1.8.0", torchvision="0.9.0", torchtext="0.9.0"),
     dict(torch="1.7.1", torchvision="0.8.2", torchtext="0.8.1"),
     dict(torch="1.7.0", torchvision="0.8.1", torchtext="0.8.0"),
```

Comment:
If we drop support for `torch<1.7.1`, should this line also be removed?
```suggestion
```

Reply:
Chose to keep it for now in case we use 1.7.0 somewhere. Using 1.7.1 is a limitation due to Python 3.9 support, not in general